package.json

```diff
@@ -21,17 +21,18 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "nyc --reporter=text --reporter=html mocha"
+    "test": "./node_modules/.bin/nyc --reporter=text --reporter=html mocha"
```

Comment:
When `nyc` is installed as a `devDependency` this change is not necessary, `npm` automatically includes the `./node_modules/.bin` in the `PATH` when executing.

Reply:
Got it. I'll fix that up now and push an update now then

Reply:
Thanks... I'm going to merge this in, then will you rebase the other PR which will get the tests to run (at least on the good node versions)?

Reply:
Yep, sounds good

Reply:
looks like it's not necessary... i was able to restart travis and it pulled in the latests package.json, thanks again!

Reply:
Ah great, thanks!